### ISSUE TITLE
Basic styling of checkbox should not require javascript initialisation

### ISSUE THREAD

**Issue by timlind** [google/material-design-lite#1891]
Without the class 'mdl-js-checkbox' before javascript initialisation, checkbox's are unstyled.

**Comment by surma** [comment]
Just for references: That is due to us *intentionally* falling back to unstyled checkboxes for IE9.

**Comment by timlind** [comment]
FYI this was on webkit.

-- 
Tim Lind
tim@incremental.co | +27827895930
Incremental Code: Internet Software Studio
<http://www.incremental.co>

**Comment by Garbee** [comment]
Yes, it happens everywhere because we need it to not style in IE9 due to
technical limitations. Once we drop support for this browser we can address
the core problem for all supported browsers.

**Comment by sgomes** [comment]
Yup, @surma and @Garbee are both correct. With V2 we'll drop IE9 support, and we'll move to styling the component without JS (for everything but the ripple, at least).
